### DISCUSSION TITLE
NVR integrabile in HA

### DISCUSSION THREAD

**Discussion by LuigiPzz** [indomus/forum#1065] [Hardware & Firmware]
Salve a tutti,
è svariato tempo che mi cimento nell'integrazione (sensata) di videocamere nella mia domotica (con Home assistant).
Il problema principale è che avendo al momento 5 cam (e volendo arrivare a 8), pur essendo le stesse di poche pretese (max 1080p 20fps), sia l'integrazione diretta, sia l'integrazione dedicata su una seconda raspberry pi 4 sempre con motioneye porta la cpu della stessa al 100% con riconoscimento disattivato.

Do per scontato che stia sbagliando qualcosa, ma non ostante ciò non credo comunque di poter arrivare ad una soluzione di 8 camere 1080p con riconoscimento attivato senza usare 2 raspberry pi4 dedicate solo a questo. (Libero di essere smentito).

Quello che chiedo è dunque, esiste un nvr che abbia la possibilità di essere integrato in home assistant?

**Comment by snakuzzo** [comment]
Ti chiedo cortesemente di utilizzare un titolo che identifichi meglio il tema. Grazie

**Comment by NdR91** [comment]
Ciao, fermorestando che con ME utilizzando uno streaming video passtrought non dovrebbe saturare eccessivamente la CPU, comunque 8 telecamere non sono poche.

Probabilmente, in questo caso, piuttosto che due RPi4 dovresti valutare un MiniPC con CPU x86 (Intel/AMD). 
Poi, se l'intento è utilizzare un sistema di riconoscimento degli oggetti, allo stato attuale la scelta migliore credo ricada su Frigate con ausilio di TPU Coral...
